consensus/src/pipeline/virtual_processor/processor.rs

```diff
@@ -769,13 +776,22 @@ impl VirtualStateProcessor {
             ));
         }
 
+        {
+            // Set the pruning point utxoset position to the new point we just verified
```

Comment:
Shouldn't we set it after we write the UTXO set?

Reply:
exactly why the function name is misleading (see TODO). At this point the pruning point utxo-set is already written, the import is *from it* to virtual utxo-set